### PR TITLE
🧪Introduce a4a-no-signing experiment diversion

### PR DESCRIPTION
**Pull request by calebcordry**
Adds diversion point, experiment will not be launched until logic is complete.

Part of #27189

### CODE REVIEW

File: extensions/amp-a4a/0.1/amp-a4a.js [hunk 1]
```diff
@@ -810,7 +820,10 @@ export class AmpA4A extends AMP.BaseElement {
         return fetchResponse;
       })
       .then((fetchResponse) =>
-        this.startValidationFlow_(fetchResponse, checkStillCurrent)
+        getExperimentBranch(this.win, NO_SIGNING_EXP.id) ===
+        NO_SIGNING_EXP.control
+          ? this.streamResponse_(fetchResponse)
```
Comment (lannka) [inline extensions/amp-a4a/0.1/amp-a4a.js:825]: should it be the reverse?

File: extensions/amp-a4a/0.1/amp-a4a.js [hunk 2]
```diff
@@ -810,7 +820,10 @@ export class AmpA4A extends AMP.BaseElement {
         return fetchResponse;
       })
       .then((fetchResponse) =>
-        this.startValidationFlow_(fetchResponse, checkStillCurrent)
+        getExperimentBranch(this.win, NO_SIGNING_EXP.id) ===
+        NO_SIGNING_EXP.control
```
Comment (powerivq) [inline extensions/amp-a4a/0.1/amp-a4a.js:824]: `NO_SIGNING_EXP.experiment` here?

File: extensions/amp-a4a/0.1/amp-a4a.js (hunk 1 above)
Comment (calebcordry) [reply extensions/amp-a4a/0.1/amp-a4a.js:825]: Yikes, good catch. No idea why the tests passed.

Comment (calebcordry) [reply extensions/amp-a4a/0.1/amp-a4a.js:825]: Oh I think its because this returns `undefined`

Comment (lannka) [reply extensions/amp-a4a/0.1/amp-a4a.js:825]: ye, actually should change control to experiment, as @powerivq pointed out in the other comment.

Comment (calebcordry) [reply extensions/amp-a4a/0.1/amp-a4a.js:825]: done.

File: extensions/amp-a4a/0.1/amp-a4a.js (hunk 2 above)
Comment (calebcordry) [reply extensions/amp-a4a/0.1/amp-a4a.js]: yep, done.

**Pull request merged**